benchmark/benchmark.py

```diff
@@ -0,0 +1,74 @@
+import time
```

Comment:
What are we benchmarking in inference toolkit? I don't think this is required in this repo.